README.md

```diff
@@ -93,3 +93,94 @@ extension BannerView: BannerRendering {
 }
 ```
 
+## 🛰 Interactor
+
+> **Interactor**
+> (*noun*)
+> 
+> Component answering the question: What should I do in reaction to user action. 
+
+### Definitions
+
+### TL;DR
+
+Interactor is a final class that ViewController owns, which performs actions in reaction to user input - gestures, text input, device movement, geographical location change etc.
+
+#### Interactor is: 
+- active component (has a lifecycle), 
+- final class (1),
+- can use other components for data retrieval etc.
+- should render methods on weakly held views.
+
+*(1) There is no good reason for inheritance of custom classes, ever.*
+
+#### ViewControllers:
+
+- can have many Interactors
```

Comment:
With that said, I'm missing a guideline / rule to determine the scope of single `Interactor`.
* Is "one view - one `Interactor`" a valid rule?
* What if `Interactors` need to talk to each other?

Reply:
I'm not sure, sometimes when views are simple and no complicated logic is in Interactor, it can handle couple of them.

If they want to talk to each other, we should probably use some Mediator to encapsulate the logic of what is going on, no direct knowledge of existence is needed between them.